x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ingest_pipelines/ingest_pipelines_card.tsx

```diff
@@ -62,7 +63,7 @@ export const IngestPipelinesCard: React.FC = () => {
           <EuiPanel color="subdued">
             <DefaultPipelineItem
               index={index}
-              openModal={openModal}
+              openModal={openPipelineSettings}
```

Comment:
Should we rename this `openModal` prop too?

Reply:
good point, done :)